scripts/variantstore/beta_docs/gvs-outputs.md

```diff
@@ -0,0 +1,126 @@
+| high_CALIBRATION_SENSITIVITY_SNP               | Sample Genotype FT filter value indicating that the genotyped allele failed SNP model calibration sensitivity cutoff (0.997)  | The VETS filtering score. This is a ##COMMENT in the VCF because this is not applied as a site-level filter. GVS is allele-specific. The FT for an allele will have this FT tag if it hits the CALIBRATION_SENSITIVITY threshold. A site can pass with failing genotypes, however we recommend filtering these genotypes. See example variants below. |
+| high_CALIBRATION_SENSITIVITY_INDEL             | Sample Genotype FT filter value indicating that the genotyped allele failed INDEL model calibration sensitivity cutoff (0.99) | The VETS filtering score. This is a ##COMMENT in the VCF because this is not applied as a site-level filter. GVS is allele-specific. The FT for an allele will have this FT tag if it hits the CALIBRATION_SENSITIVITY threshold. A site can pass with failing genotypes, however we recommend filtering these genotypes. See example variants below. |
+| EXCESS_ALLELES                                 | Site has an excess of alternate alleles based on the input threshold                                                          | When a site has more than 100 alternate alleles, this filter will be present.                                                                                                                                                                                                                                                                         |
+| OUTSIDE_OF_TARGETS                             | Outside of sequencing target intervals                                                                                        | Exome only. The site is not within the target intervals of the exome assay. We recommend filtering out these sites as we cannot stand behind the quality of sites called outside of the targets.                                                                                                                                                      |
+
+
+### Example sites and how to interpret them
+#### Passing site
+```
+chr20   66369   .       C       T       .       .       AC=2;AF=1.00;AN=2;AS_QUALapprox=0|982;AS_YNG=G;CALIBRATION_SENSITIVITY=0.2492;QUALapprox=982;SCORE=-0.3836      GT:AD:GQ:RGQ    ./.         ./.     1/1:0,29:87:982
+```
+
+#### A site with failing calibration sensitivity SNPs for all sample genotypes
+```
+chr20   62898   .       T       A       .       .       AS_QUALapprox=0|933;AS_YNG=G;CALIBRATION_SENSITIVITY=0.9999;QUALapprox=324;SCORE=-0.7456        GT:AD:FT:GQ:PGT:PID:RGQ 0/1:26,11:high_CALIBRATION_SENSITIVITY_SNP:99:0|1:62818_T_G:384     0/1:23,7:high_CALIBRATION_SENSITIVITY_SNP:99:0|1:62818_T_G:225  0/1:18,9:high_CALIBRATION_SENSITIVITY_SNP:99:0|1:62818_T_G:324
```

Comment:
So if everybody fails there's just no AC/AN?

Reply:
Yes, we don't calculate it when every genotype fails.

Reply:
Following up from our call - the reason this doesn't have NO_HQ_GENOTYPES site level filter is because the GQ for each allele is high.